package-lock.json

```diff
@@ -87,7 +87,7 @@
         "debug": "^4.1.0",
         "gensync": "^1.0.0-beta.1",
         "json5": "^2.1.2",
-        "lodash": "^4.17.20",
+        "lodash": "^4.17.19",
```

Comment:
Didn't 4.17.19 have the security vulnerability?

Reply:
It passes the synk test so I guess it must be ok? 🤷‍♀️

Reply:
That indicates that the dependency _requires_ 4.17.19 and up when you run `npm ls lodash` the output indicates lodash 4.17.20 has been resolved: 
![image](https://user-images.githubusercontent.com/9645462/92725502-28805680-f364-11ea-9ca2-a8fc1895ef51.png)